Include/internal/pycore_code.h

```diff
@@ -24,9 +27,117 @@ struct _PyOpcache {
     char optimized;
 };
 
+
+// XXX Can we do this with an enum instead?
+typedef unsigned char _PyFastLocalKind;
+#define CO_FAST_POSONLY     0x01
+#define CO_FAST_POSORKW     0x02
+#define CO_FAST_VARARGS     0x04
+#define CO_FAST_KWONLY      0x09
```

Comment:
9? Really? Why not a power of 2 like all the others?

Reply:
That's a typo.  Good catch!